web.browser/src/actions/fetchActions.js

```diff
@@ -0,0 +1,40 @@
+export const LOADPOSTS = 'LOADPOSTS';
+const LOADCATEGORIES = 'LOADCATEGORIES';
+export const LOADWEEKS = 'LOADWEEKS';
+
+export function loadPosts(items) {
+  return {
+    type: LOADPOSTS,
+    payload: items,
+  };
+}
+export function loadCategories(items) {
+  return {
+    type: LOADCATEGORIES,
+    payload: items,
+  };
+}
+export function loadWeeks(items) {
```

Comment:
It's interesting that you've mixed ES5 function declarations w/ ES6 function syntax in your fns below. Consistency is preferred.